.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: release
+
+on:
+  push:
+    branches:
+      - [main]
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Is this necessary? It is slower than the default of only fetching HEAD.

Reply:
That's necessary to compute the next release version when the release library hasn't been installed before.